app/assets/stylesheets/components/_markdown-editor.scss

```diff
@@ -7,13 +7,15 @@ $app-toolbar-button-size: 50px;
 
     &:focus {
       outline: none;
+      box-shadow: none;
     }
   }
 }
 
 .app-c-markdown-editor__container--focused {
   outline: $govuk-focus-width solid $govuk-focus-colour;
   outline-offset: 0;
+  box-shadow: inset 0 0 0 2px;
```

Comment:
From this I'm getting that we're using box-shadow instead of outline?

Reply:
Yes, that is right.